docs/versions/v26.4.5/guide/evolve/reference-implementation.md

```
@@ -0,0 +1,281 @@
+- Download the configuration file
+- Use it with the template generator:
+
+```bash
+java -jar template-generator-1.0.0.jar -c downloaded-config.yaml -o ./my-pipeline-app
+```
+
+## Generated Project Structure
+
+The template generator creates a complete Maven multi-module project:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)